internal/driver/driver.go

```diff
@@ -633,23 +634,31 @@ func (d *Driver) startStreaming(device *Device) errors.EdgeX {
 		return errors.NewCommonEdgeX(errors.KindServerError, fmt.Sprintf(
 			"failed to start video streaming for device %s", device.name), err)
 	}
+	startErrs := make(chan errors.EdgeX)
```

Comment:
Always have buffer of at least one
```suggestion
	startErrs := make(chan errors.EdgeX, 1)
```